### DISCUSSION TITLE
Tests are not running at non headless mode at Github Actions

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [selenide/selenide#2335] [Q&A]
Hi, I have automation tests written in Selenide & Java, I can't run them in non headless mode at Github actions, here is an error I get:

```
Could not start a new session. Response code 500. Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.) 
Host info: host: 'fv-az357-439', ip: '10.1.0.175'
Build info: version: '4.9.1', revision: 'eb2032df7f'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-1038-azure', java.version: '11.0.19'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, newSession {capabilities=[Capabilities {acceptInsecureCerts: true, browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*, --proxy-bypass-list=<-loopb..., --disable-dev-shm-usage, --window-size=1366,768], excludeSwitches: [enable-automation, load-extension], extensions: [], prefs: {credentials_enable_service: false, download.default_directory: /home/runner/work/qa-automa..., plugins.always_open_pdf_externally: true, profile.default_content_setting_values.automatic_downloads: 1, safebrowsing.enabled: true}}, pageLoadStrategy: normal, unhandledPromptBehavior: accept}]}]
```
when I will put headless=true - everything work, but if headless=false the problem above appears. Locally tests work in any variety, issue reproduce only at Github actions, pls help me to solve it?

FYI I use chrome options, like this:
`--no-sandbox,--disable-gpu,--disable-dev-shm-usage, --remote-allow-origins=*, --user-data-dir=~/.config/google-chrome
`but it doesn't help.

**Comment by VladyslavZagorodnyi** [comment]
@asolntsev Pls help to figure out what can be a root cause of it.

**Comment by asolntsev** [comment]
@VladyslavZagorodnyi This is an expected behaviour. Browser is run on a Linux server which doesn't have graphical environment. It cannot work in non-headless mode.

However, you have few options to overcome this problem:
1. Run browser in a Docker (see "Selenoid" etc.)
2. In Github actions configuration, choose Windows or Mac OS instead of Ubuntu.
